pages/mesosphere/dcos/services/spark/2.9.0-2.4.3/release-notes/index.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 10
+excerpt: Release notes for DC/OS Apache Spark 2.9.0-2.4.3
+featureMaturity:
+render: mustache
+model: /mesosphere/dcos/services/spark/data.yml
+---
+
+## Version Spark and Spark history 2.9.0-2.4.3
+
+### New features
+- Marathon group role enforcement support for DCOS 1.14
+- Role propagation from Dispatcher to all submitted Spark applications
+- Dispatcher role enforcement for submitted Spark applications
```

Comment:
Did we add any new docs to explain these features in our service guide? Would be good to have some information/examples so users know what they need to submit to enable new features.

Reply:
Let's add node draining to fault-tolerance section (under Driver) and quota to installation section? WDYT?